app/services/decorators/v5/property_decorator.rb

```diff
@@ -19,7 +19,7 @@ def payload
           main_home: @record.main_home,
           shared_with_housing_assoc: @record.shared_with_housing_assoc,
```

Comment:
Maybe this is outside the scope of this fix, but I strongly suspect we'll want to fix the CCQ results page so it continues to show our workings correctly in the property sub-totals tables. For that to work I think we'll need to return, for each property, how much smod was applied in this bit of the payload. Is that worth adding here?

Reply:
Yes I agree. Myself and Mike H had the same discussion this morning. I think its another piece of work